src/libraries/System.IO.FileSystem/tests/File/GetSetTimes.cs

```diff
@@ -1,21 +1,21 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Collections.Generic;
-using System.Linq;
-using System.Runtime.InteropServices;
 using System.Threading.Tasks;
 using Xunit;
 
 namespace System.IO.Tests
 {
-    public class File_GetSetTimes : StaticGetSetTimes
+    public abstract class File_GetSetTimes : StaticGetSetTimes
     {
-        protected override bool CanBeReadOnly => true;
-
         // OSX has the limitation of setting upto 2262-04-11T23:47:16 (long.Max) date.
         // 32bit Unix has time_t up to ~ 2038.
-        private static bool SupportsLongMaxDateTime => PlatformDetection.IsWindows || (!PlatformDetection.Is32BitProcess && !PlatformDetection.IsOSXLike);
+        protected static bool SupportsLongMaxDateTime => PlatformDetection.IsWindows ||
+                                                       !PlatformDetection.Is32BitProcess &&
+                                                       !PlatformDetection.IsOSXLike;
+
+        protected override bool CanBeReadOnly => true;
```

Comment:
nit: it would be nice to limit re-ordering members, such changes clutter up the diff.